.github/workflows/python-wrapper-unit-tests.yaml

```diff
@@ -9,20 +9,23 @@ jobs:
   paths-filter:
     runs-on: ubuntu-22.04
     outputs:
-      client-change: ${{ steps.filter.outputs.client == 'true' }}
+      wrapper-change: ${{ steps.filter.outputs.wrapper == 'true' }}
+      sdk-change: ${{ steps.filter.outputs.sdk == 'true' }}
     steps:
-      - uses: actions/checkout@v3
-      - uses: dorny/paths-filter@v2
+      - uses: actions/checkout@v4
+      - uses: dorny/paths-filter@v3
         id: filter
         with:
           filters: |
-            client: 
+            wrapper: 
               - 'clients/python-wrapper/**'
               - '.github/workflows/python-wrapper-unit-tests.yaml'
+            sdk:
+              - 'clients/python/**'
 
   always-succeed:
     needs: paths-filter
-    if: ${{ needs.paths-filter.outputs.client-change != 'true' }}
+    if: ${{ needs.paths-filter.outputs.wrapper-change != 'true' }}
```

Comment:
always succeed when both wrapper and sdk-change are unchanged

Reply:
Not sure I follow - this job is meant to override the one with the same name in case we didn't change anything in the wrapper because this job is a required one. The new job added is not required for merging - it's optional as explained in the PR description

Reply:
got it. thanks.